src/Frontend/Page/Idea.hs

```diff
@@ -169,6 +169,13 @@ linkToIdeaLocation idea = do
              IdeaLocationSpace{} -> "Zum Ideenraum"
              IdeaLocationTopic{} -> "Zum Thema"
 
+-- ASSUMPTION: The caps list does con contain the same value marked as
+-- Clickable and GrayedOut
```

Comment:
Also, I'm wondering if there is a better name?  `canGrayOut`?  No, that's no better.  I appreciate the rule that eliminators are called like the type, `maybe` does it, too. I just still find it hard to read.